auth/api/http/orgs/transport.go

```diff
@@ -241,6 +246,30 @@ func decodeListGroupsRequest(_ context.Context, r *http.Request) (interface{}, e
 	return req, nil
 }
 
+func decodeCreatePoliciesRequest(_ context.Context, r *http.Request) (interface{}, error) {
+	if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
+		return nil, apiutil.ErrUnsupportedContentType
+	}
+
+	req := createPoliciesReq{
+		token:   apiutil.ExtractBearerToken(r),
+		orgID:   bone.GetValue(r, orgIDKey),
+		groupID: bone.GetValue(r, groupIDKey),
+	}
+
+	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
+		return nil, errors.Wrap(apiutil.ErrMalformedEntity, err)
+	}
+
+	for _, gp := range req.GroupPolicies {
+		if gp.Policy == "" {
+			gp.Policy = auth.RPolicy
+		}
+	}
```

Comment:
This is unecessary, no need to fix a wrong request. If policy is empty an error will be returned in the validation.